tern/formats/generator.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (c) 2019 VMware, Inc. All Rights Reserved.
+# SPDX-License-Identifier: BSD-2-Clause
+
+import abc
+
+
+class GenerateReport(object):
+    '''Base class for report plugins'''
+    def __init__(self):
+        '''Do nothing'''
+    @abc.abstractmethod
+    def generate(self, image_obj_list):
+        '''Format the report according to the plugin style'''
+        '''Each subclass is responsible for their own formatting.'''
```

Comment:
Prospector didn't complian about the "pointless string statement" but you can run these statements together and then put `pass` at the end. See `tern/classes/template.py` as an example.

Reply:
Fixed in the updated commit.